models/articles.model.js

```diff
@@ -23,7 +23,18 @@ exports.getAllArticles = () => {
         ORDER BY created_at DESC;`
     )
     .then((data) => {
-      console.log(data.rows)
       return data.rows;
     });
 };
+
+exports.fetchCommentsByArticleId = (article_id) => {
+  return db
+  .query(`SELECT comment_id, comments.votes, comments.created_at, comments.author, comments.body, comments.article_id FROM comments
+          FULL OUTER JOIN articles
```

Comment:
im not sure you need a join for this, you could probably just go into the comments table and get the ones with the article if you're after. 

Maybe dont change it now as it seems to pass, but jsut something to think about